components/arm/xarm/xarm7_kinematics.json

```diff
@@ -61,16 +61,16 @@
                 "y": 130,
                 "z": 180,
                 "translation": {
-                    "x": 0,
+                    "x": 32.880465325174462520863017,
                     "y": -10,
-                    "z": 46.5
+                    "z": 32.880465325174455415435659
```

Comment:
Note: specifying the translation of these geometries down to rontometers (10^-27) expresses a lot of optimism about humanity's understanding of the nature of distance.

Reply:
😅 will edit to 10^-2